src/index.ts

```diff
@@ -30,11 +31,7 @@ async function evaluate(testSuite: EvaluateTestSuite, options: EvaluateOptions =
   return ret;
 }
 
-module.exports = {
-  evaluate,
-  assertions,
-  providers,
-};
+export { evaluate, assertions, providers };
```

Comment:
This change technically unrelated, but allows parity with how we'd export `generateTable`. It also fixed an issue where I ended up having to import like this:

```ts
import {default as promptfoo} from 'promptfoo';
```

With this change, I think the following should work in my setup:

```ts
import {evaluate} from 'promptfoo';
```

Open to reverting this though if you see it as a problem.